unfoldingWord/translationCore
# License Page

#### This pull request addresses:

Addresses https://github.com/unfoldingWord-dev/translationCore/issues/967



#### How to test this pull request:

Click Home in the top left, click on the licenses tab, the information should display there.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1215)
